internals/secrethub/service_ls_test.go

```diff
@@ -57,6 +66,62 @@ func TestServiceLsCommand_Run(t *testing.T) {
 			},
 			out: "test\nsecond\n",
 		},
+		"success aws": {
+			cmd: ServiceLsCommand{
+				serviceTable: awsServiceTable{},
+			},
+			serviceService: fakeclient.ServiceService{
+				Lister: fakeclient.RepoServiceLister{
+					ReturnsServices: []*api.Service{
+						{
+							ServiceID:   "test",
+							Description: "foobar",
+							Credential: &api.Credential{
+								Type: api.CredentialTypeAWSSTS,
+								Metadata: map[api.CredentialMetadataKey]string{
+									api.CredentialMetadataAWSRole:   "arn:aws:iam::123456:role/path/to/role",
+									api.CredentialMetadataAWSKMSKey: "arn:aws:kms:us-east-1:123456:key/12345678-1234-1234-1234-123456789012",
+								},
+							},
+						},
+					},
+				},
+			},
+			out: "ID    DESCRIPTION  ROLE                                   KMS KEY\ntest  foobar       arn:aws:iam::123456:role/path/to/role  arn:aws:kms:us-east-1:123456:key/12345678-1234-1234-1234-123456789012\n",
+		},
+		"success aws filter": {
+			cmd: ServiceLsCommand{
+				serviceTable: awsServiceTable{},
+				filters: []func(*api.Service) bool{
+					isAWSService,
+				},
+			},
+			serviceService: fakeclient.ServiceService{
+				Lister: fakeclient.RepoServiceLister{
+					ReturnsServices: []*api.Service{
+						{
+							ServiceID:   "test",
+							Description: "foobar",
+							Credential: &api.Credential{
+								Type: api.CredentialTypeAWSSTS,
+								Metadata: map[api.CredentialMetadataKey]string{
+									api.CredentialMetadataAWSRole:   "arn:aws:iam::123456:role/path/to/role",
+									api.CredentialMetadataAWSKMSKey: "arn:aws:kms:us-east-1:123456:key/12345678-1234-1234-1234-123456789012",
+								},
+							},
+						},
+						{
+							ServiceID:   "test2",
+							Description: "foobarbaz",
+							Credential: &api.Credential{
+								Type: api.CredentialTypeRSA,
+							},
+						},
+					},
+				},
+			},
+			out: "ID    DESCRIPTION  ROLE                                   KMS KEY\ntest  foobar       arn:aws:iam::123456:role/path/to/role  arn:aws:kms:us-east-1:123456:key/12345678-1234-1234-1234-123456789012\n",
```

Comment:
The KMS KEY ID is basically an ARN too? Does the UUID `12345678-1234-1234-1234-123456789012` suffice here?

Reply:
I'd think the account and region are important as well.

@jpcoenen What'd be best to display here?

Reply:
Other places we use just the ID.

Reply:
Ahh, that's actually also what is currently returned :+1: Just mocking this wrong.
Should we rename the metadata key to `kms_key_id` to make this clearer?

Do we want to add the role as another credential metadata entry then?

Reply:
> Should we rename the metadata key to kms_key_id to make clearer what's returned?

If the ID is what it is, then certainly yes make it `kms_key_id` 👍 

> Do we want to add the region as another credential metadata entry then?

That seems like a very good candidate to add yes 👍